testbeam_analysis/gui/dut_types.yaml

```diff
@@ -0,0 +1,12 @@
+!!python/unicode 'FE-I4':
```

Comment:
The `+!!python/unicode` is not needed an should be deleted. It is `yaml` and not a python script.